hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/config/MirrorImporterConfiguration.java

```
@@ -103,6 +110,25 @@ AwsCredentialsProvider awsCredentialsProvider(CommonDownloaderProperties downloa
         return awsCredentials;
     }
 
+    @Bean
+    FlywayConfigurationCustomizer flywayConfigurationCustomizer() {
+        return configuration -> {
+            Long timestamp = mirrorProperties.getTopicRunningHashV2AddedTimestamp();
+            if (timestamp == null) {
+                if (mirrorProperties.getNetwork() == HederaNetwork.MAINNET) {
+                    throw new IllegalArgumentException("topicRunningHashV2AddedTimestamp is unknown for MAINNET. To " +
```

I would prefer we hardcode the mainnet time for mainnet deploy that is currently scheduled on status.hedera.com for Thursday, May 21st, at 17:00 UTC. We can always update it later if it changes.

We wanted it to fail fast on deploy to mainnet, but the version that we will actually deploy to mainnet that runs this patch will be 0.10.1 anyway and that will fail fast if this is not populated. So we are fine with a possibly inaccurate timestamp for mainnet for 0.12.x.

---

Am not worried about us. We know all about it.
Just that, I don't want anyone to have corrupt data if they upgrade from before 0.10.0  to 0.12.x. Also BRD has its wanted fix in 0.12.x too. 
As long as we put right timestamp in 0.12.0, even if it means delaying release a couple days, it's good.